src/db/migrations/20210319133351-add-featured-to-stalls.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = {
```

Comment:
Have you tried this migration with existing stalls in the database that does not have the `isFeatured` flag?

Reply:
This is quite impt!

Reply:
yes I've tried and and the default is set to false